src/patternfly/components/OptionsMenu/options-menu-multiple.hbs

```diff
@@ -15,7 +15,7 @@
     {{/options-menu-menu-item}}
   </ul>
 </li>
-{{#> divider divider--type="li"}}{{/divider}}
+{{#> options-menu-separator}}{{/options-menu-separator}}
```

Comment:
Background color should be the same as the separator in the dropdown etc. 

<img width="273" alt="Screen Shot 2019-10-23 at 10 44 59 AM" src="https://user-images.githubusercontent.com/20118816/67405081-46916c80-f582-11e9-81b4-63ac0c925f30.png">

Reply:
updated